src/main/webapp/css/main.css

```diff
@@ -14,45 +14,83 @@
  */
```

Comment:
The styles here apply on all pages, along with any specific css files that get loaded only on given pages